src/coord/src/catalog/storage.rs

```diff
@@ -92,6 +92,14 @@ const MIGRATIONS: &[&str] = &[
         name TEXT PRIMARY KEY,
         value TEXT
     );",
+    // Creates the roles table and a default "materialize" user.
+    //
+    // Introduced in v0.6.2.
+    "CREATE TABLE roles (
```

Comment:
Would it be worth preemptively adding the other fields to `roles` to avoid writing another migration later? (I'm guessing we'll need `login` and `superuser` at some point.)

Reply:
Ah yeah, I think it's as likely that we'll get the columns we need wrong if we add them today as it is that we get them right. And SQLite only supports `ALTER TABLE ... ADD COLUMN`—schema changes that remove columns are a [giant pain](https://sqlite.org/lang_altertable.html#making_other_kinds_of_table_schema_changes). So I figure it's best to wait until we're sure we need those columns!